### DISCUSSION TITLE
Migrating to a GitHub organization

### DISCUSSION THREAD

**Discussion by ORelio** [ORelio/Minecraft-Console-Client#1755] [Announcements]
Hi everyone!

More than 9 years after [starting this project](http://www.minecraftforum.net/topic/1314800-/), I'm glad to see that Minecraft Console Client has its own community 🥇

To help MCC get better maintained on the long term, I think it would be best to move to a GitHub organization. The main reason behind this is that GitHub organizations allow [much more flexibility in permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level) than [personal project permissions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/permission-levels-for-a-user-account-repository). This would allow granting the same level of access than myself to trusted maintainers, and giving some access to help managing issues.

Before changing anything, feel free to use this thread to make any suggestion, such as what name we could take for our organization, what changes we could make to better manage the repository, etc. 😃

**Comment by ORelio** [comment]
How about "MCCTeam" as an organization name? Simple but self-explaining.

**Reply by ReinforceZwei** [reply]
I'm okay with this name :smile:

**Reply by ORelio** [reply]
I think we had well enough time for people to make comments, so I'll create the organization soon.

**Reply by ORelio** [reply]
Migration done!
